### PR TITLE
Add bulk issue to case-worker-guide

### PR DESCRIPTION
**Pull request by whaught**
Fixes https://github.com/google/exposure-notifications-verification-server/issues/1250

## Proposed Changes

* Add details about bulk-upload to the user guide

**Release Note**

```release-note
Added user-guide documentation on bulk code issuance.
```

### CODE REVIEW

File: docs/case-worker-guide.md [hunk 1]
```diff
@@ -40,3 +40,35 @@ After the code is successfully issued, the `short code` will be displayed to be
...
+![bulk issue menu](images/users/menu_bulk_issue.png "bulk issue menu")
+
+This allows the user to upload a .csv file and issue many codes at once to a list of patient phone numbers and their associated test date. The bulk uploader assumes that all results are `positive`.
+
+The bulk uploader is written as a javascript client. Rather than uploading the file, it parses on the client and makes requests to the issue API. This allows the server to discard phone numbers after sending the SMS without storing them. The uploader may be canceled and resumed, respects the server's rate-limiting throttle, and prevents the user from sending an SMS to the same phone twice using a retry code.
+
+### CSV Format
+`patient phone`,`test date`, [optional] `symptom date`
+
+* The patient phone should be in [E.164 format](https://www.twilio.com/docs/glossary/what-e164).
```
Comment (mikehelmick) [inline docs/case-worker-guide.md:57]: Replace should with must / next line too

Comment (whaught) [reply docs/case-worker-guide.md]: done

**Pull request merged**